WordPress/UITests/Utils/XCTest+Extensions.swift

```diff
@@ -85,4 +99,56 @@ extension XCTestCase {
         static let category = "iOS Test"
         static let tag = "tag \(Date().toString())"
     }
+
+    private func fetchScenarios(completion: @escaping ([String: Any]?, Error?) -> Void) {
+        guard let url = URL(string: "http://localhost:8282/__admin/scenarios") else {
+            completion(nil, NSError(domain: "InvalidURL", code: -1, userInfo: nil))
+            return
+        }
```

Comment:
Feel free to [force unwrap](https://docs.swift.org/swift-book/documentation/the-swift-programming-language/thebasics#If-Statements-and-Forced-Unwrapping) the result here.

(Update: Only after posting I noticed that the method below uses this approach already. I'm leaving my rationale for why it's okay here for reference).

The URL we use here is hard-coded and defined by us, the developers. It's unlikely to not work—and if it doesn't, it's appropriate to crash.

```suggestion
        let url = URL(string: "http://localhost:8282/__admin/scenarios")!
```

Reply:
Another note, the base URL part could come from [`WireMock`](https://github.com/wordpress-mobile/WordPress-iOS/blob/0ac167921891bd904d358e7472be65fbe757c308/WordPress/UITestsFoundation/WireMock.swift), so that there's a single source of truth and we can also change it via launch environment.

```suggestion
         let url = URL(string: "\(WireMock.URL())__admin/scenarios")!
```

_Side note_ I think if we do this, it would be best to remove the trailing `/` from `URL()` and have it here for clarity. I assume there's no specific reason to have it in the base URL, but I could be wrong.